modules/media/test/TestPlan.md

```diff
@@ -0,0 +1,95 @@
+## Media Module Test Plan
+
+### 📄 Overview
+
+Media module allows users to upload, browse and edit media files associated with a specific timepoint in Loris.
+
+### 🔒 Permissions
+
+In order to use media module user might need one or both of the following permissions:
```

Comment:
In order to use **the** media module**, the** user might need one or both of the following permissions: